web_pwa_oca/static/src/js/worker/pwa.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+/* eslint-disable no-undef, no-empty-function, no-implicit-globals, no-unused-vars */
+/* Copyright 2020 Tecnativa - Alexandre D. Díaz
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+
+/**
+ * Services workers are a pice of software separated from the user page.
+ * Here can't use 'Odoo Bootstrap', so we can't work with 'require' system.
+ * When the service worker is called to be installed from the "pwa_manager"
+ * this class is instantiated.
+ */
+var PWA = OdooClass.extend({
+
+    init: function () {
+        // To be overrided
```

Comment:
```suggestion
        // To be overriden
```

Reply:
`Overridden`, to be absolutely correct in English :-)